### PR TITLE
owo with a probability

### PR DESCRIPTION
**Pull request by kragniz**
entirely untested

### CODE REVIEW

**Comment by zuzak** [comment]
Do you care enough to add[ validation to the config variable](https://github.com/mozilla/node-convict#custom-format-checking)

**Comment by kragniz** [comment]
not really

File: modules/news.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ module.exports = {
             } else {
               str += news.text
             }
-            if (bot.config.get('news.owo')) {
+            if (Math.random() >= bot.config.get('news.owo')) {
```
Comment (zuzak) [inline modules/news.js]:
```suggestion
            if (Math.random() <= bot.config.get('news.owo')) {
```

**Comment by kragniz** [comment]
I like this better

**Pull request merged**